README.md

```diff
@@ -8,4 +8,11 @@ To link Database files, create .env.test and .env.development files with the rel
 
 PGDATABASE=database_name_here
 
-Ensure these files are included in your .gitignore
\ No newline at end of file
+Ensure these files are included in your .gitignore
+
+## Run database
+
+Run your PSQL file :
+
+    psql -f db/setup.sql
```

Comment:
you have scripts to run this, you can tell them to use that instead of the file!